master/internal/db/postgres_cluster.go

```diff
@@ -32,6 +32,38 @@ WHERE NOT EXISTS ( SELECT * FROM cluster_id );
 	return uuidVal[0], nil
 }
 
+func (db *PgDB) UpdateClusterHeartBeat(current_cluster_heartbeat time.Time) error {
+	var uuidVal []string
+	if err := db.sql.Select(&uuidVal, `SELECT cluster_id FROM cluster_id`); err != nil {
+		return errors.Wrapf(err, "error reading cluster_id from cluster_id table")
+	}
+	if len(uuidVal) != 1 {
+		return errors.Errorf(
+			"expecting exactly one cluster_id from cluster_id table, %d values found", len(uuidVal),
+		)
+	}
```

Comment:
this is all basically unused, I think we can just get rid of this. if it is to check the 1 row invariant is upheld, then i would say the check should happen elsewhere because there is nothing we can do about it here, we may as well still do what we need to do (set cluster heartbeat).

Reply:
I agree. done!